manifest-spec.md

```diff
@@ -1,37 +1,140 @@
+
+## Manifest format
+
+The canoonical format for the package manifest is a JSON document containing a
+single JSON object.  
+
+
+## Manifest filename
+convention is to name this document `epm.json` which is short for *'ethereum
+package manifest'*.
+
+## Manifest Data Overview
+
+The following fields are defined as data that *may* be included in a package
+manifest.  Custom fields should be prefixed with `x-` to prevent collision with
+new fields introduced in future versions of the specification.
+
+- manifest version
 - package name
 - author
-- version (preferably [semver](http://semver.org/))
+- version
 - description
 - contract metadata
   - abi
   - unlinked binary
-  - deployed address if applicable
-  - name/address pairs for linked libraries
+  - deployed addresses
+  - linked libraries
   - compiler version for bytecode verification
-  - key/value pairs of the sha3 hash of events this contract can emit with the abi definition of that event (i.e., the contract metadata should contain information for all events that can be triggered by this contract, including those triggered by linked libraries)
 - external project URIs (homepage, repository uri, etc.)
 - dependencies (i.e., references to other packages and versions)
-- updated time (likely added by the packager)
 
+## Manifest Data Specification
+
+### Manifest Version: `manifest_version`
+
+
+The `manifest_version` field defines the version of the ethereum package manifest
+specification (this document) that this manifest conforms to. All manifests
+**must** include this field.
+
+* Key: `manifest_version`
+* Type: Integer
+* Allowed Values: `1`
+* Package Manager Guide: Package managers should validate this field is valid prior to publishing new manifests.
+
+
+### Package Name: `package_name`
+
+The `package_name` field defines a human readable name for this package.  All
+manifests **should** include this field.
+
+* TODO: what should happen if this is missing (should it be required?)
+* Key: `package_name`
+* Type: String
+* Package Manager Guide: Package managers should use this field when registering new packages (TODO: define this better)
+
+### Author: `author`
+
+The `author` field defines a human readable name for the author of this package.  All
+manifests **should** include this field. 
 
-## Manifest format
 
-Package Manifests can be stored in the following formats:
+* Key: `author`
+* Type: String
 
-* JSON document in a `package.json` file.
+### Version: `version`
+
+The `version` field declares the version number of this release.  This value
+**should** be conform to the [semver](http://semver.org/) version numbering
+specification.  All manifests must **include** this field.
+
+* Key: `version`
+* Type: String
+* Package Manager Guide: Package Managers should validate this field conforms to the *semver* format and display a warning if it does not.
+
+
+### Description: `description`
```

Comment:
I'd suggest to add tags/keywords additionally to the description. Tags are the new links in the semantic web. Also every scientific paper provides keywords along the abstract.